llama/convert.py

```diff
@@ -44,9 +44,13 @@ def map_torch_to_mlx(key, value):
     parser = argparse.ArgumentParser(description="Convert Llama weights to MLX")
     parser.add_argument("torch_weights")
     parser.add_argument("output_file")
+    parser.add_argument('--cpu', '-c', action='store_true', help='Use CPU instead of GPU')
     args = parser.parse_args()
 
-    state = torch.load(args.torch_weights)
+    if args.cpu:
+        state = torch.load(args.torch_weights, map_location=torch.device('cpu'))
+    else:
+        state = torch.load(args.torch_weights)
```

Comment:
I would just make the `map_location=torch.device('cpu')` the default and only behavior, rather than making this an option.